docs/platform-parts/container/container-integration.md

```diff
@@ -216,18 +214,28 @@ To add a container using Carthage:
    carthage bootstrap --no-build --platform ios
    ```
 
-##### Manually adding a container
-
-To manually add a container:
+**Use git to clone container**
 
-1. Clone the container to `<Your-WorkSpace>`.
+1. Clone the container to your mobile application's project directory.
```

Comment:
Not sure what is meant by "clone to your mobile application's project directory" - Does it mean clone, then _copy_? The mobile app is very likely already under source control, so cloning another repo _into_ the project directory is not possible.